decidim-accountability/config/locales/en.yml

```diff
@@ -51,9 +51,29 @@ en:
           results: Results
         import_results:
           new:
+            download_export: Download the Export with CSV format
             import: Import
-            info: "<p>We recommend that you follow these steps:</p><ol><li>Create the Statuses for the Results that you want to add (<a href='%{link_new_status}' target='_blank'>link</a>)</li><li>Create at least one Result manually through this Admin panel (<a href='%{link_new_result}' target='_blank'>link</a>) before using Import, for having a better understanding of the format and what you'll need to fill out.</li><li>Download the Export with CSV format (%{link_export_csv})</li><li>Make the changes locally. You can only change the following columns of the CSV:</li></ol><ul><li><b>category/id:</b> ID for the Category</li><li><b>scope/id:</b> ID for the Scope</li><li><b>parent/id:</b> ID of the parent (for related Results). Optional</li><li><b>title/I18N:</b> Title on X language</li><li><b>description/I18N:</b> Description on X language</li><li><b>start_date:</b> date when the result starts execution (format YYYY-MM-DD)</li><li><b>end_date:</b> date when the result ends execution (format YYYY-MM-DD)</li><li><b>status/id:</b> ID of the Status for this result</li><li><b>progress:</b> Percentage (from 0 to 100) of the execution</li><li><b>proposals_ids:</b> internal ID of the related proposals (separated with a comma). It gets automatically converted to <span class='attribute-name'>proposal_url</span></li></ul>"
-            link: link
+            info: |
+              <p>We recommend that you follow these steps:</p>
+              <ol>
+              <li><a href='%{link_new_status}' target='_blank'>Create the Statuses for the Results</a> that you want to add</li>
+              <li><a href='%{link_new_result}' target='_blank'>Create at least one Result manually</a> through this Admin panel before using Import, for having a better understanding of the format and what you'll need to fill out.</li>
+              <li>%{link_export_csv}</li>
+              <li>Make the changes locally. You can only change the following columns of the CSV:
+                <ul>
+                <li><b>category/id:</b> ID for the Category</li>
+                <li><b>scope/id:</b> ID for the Scope</li>
+                <li><b>parent/id:</b> ID of the parent (for related Results). Optional</li>
+                <li><b>title/I18N:</b> Title on X language</li>
```

Comment:
I think the `/I18N` and the `on X language` would not be clear for any "normal" admin user trying to do the import.

Preferrably I would show all the locales that are configured for the system in this label but I can also see that becoming technically hard to implement, so maybe it would be better just to have e.g. `title/en` in the English translation indicating the English title.

Reply:
About the current approach, I remember that I thought the same the first time that I saw this. I've changed it at 1f91267eda7de075ab48a4c3589f1279c5b8da33, hopefully it's a bit cleaner.

I agree that ideally we'd have it for the languages of the Organization, but I don't see a clean way to do it at now.

Reply:
Thanks! I think it's better as it is right now. Even if it doesn't include all the languages, I believe it will be much more understandable now.